src/MTGCore/Models/Card.cs

```diff
@@ -9,7 +10,8 @@ public class Card
 
 
         public string name { get; set; }
-        public string manaCost { get; set; }
+        private string _manaCost;
+        public string manaCost { get { return _manaCost; } set { _manaCost = ManaConvert.String(value); } }
```

Comment:
this is a good solution however manacost is now entirely dependent on the manaconvery class. 

Maybe look into injecting dependancy into accessors? (dunno if this is possible)

Reply:
https://softwareengineering.stackexchange.com/questions/360525/dependency-injection-vs-static-methods

ignore me, static method id just fine. I would make a unit test tho

Reply:
I've changed my mind, making manacost dependant on the manaconvert method is spooky to me. There for i would recommend putting this method in its own service class and constructor injecting it into the controller.

you can then pass in the model and it will return a new model with a new manacost property already populated.

//this wwould be in the controller
var cardsWithSymbols = MyService.ConvertMana(cards);

Return View(cardswithSymbols)

//iinside card/cards controller wwe add new property
public string manaCostWithSymbols {get;set;}

This also means wwe keep the manacost string if we ever needed it for anything and the cards/card controller now isnt dependant on thatmethod to run.

there is plenty of documentation on DI in .net core but if you need help let me know, this would be really good practice for DI.

Reply:
I'll have a look at implementing it using this approach when I have a chance.